client/store.js

```diff
@@ -55,6 +55,16 @@ store.getCurrentAccountId = function() {
     return data.currentAccountId;
 }
 
+store.getDefaultAccountId = function() {
+    return data.settings.get('defaultAccountId');
+}
+
+store.setDefaultAccountId = function(id) {
+    //Do not use store.changeSetting so that the change is immediately visible
```

Comment:
Like here: https://github.com/bnjbvr/kresus/blob/incoming/client/store.js#L299-L302
This forwards the event so that components that subscribed to the state can get their callback triggered.